### ISSUE TITLE
Remove Roslyn integration test Roslyn workaround in tooling

### ISSUE THREAD

**Issue by NTaylorMullen** [aspnet/AspNetCore#12268]
I added a hack to workaround limitations in how the AspNetCore-Tooling's integration tests deploy their apps with a dependency on the current SDK's Roslyn (not the one the repo consumes).

If time permits in preview8 we should do the right thing and build infrastructure to properly allow the integration tests to use latest Roslyn.

At the very least we need to revert a portion of what was done [here](https://github.com/aspnet/AspNetCore-Tooling/pull/594#discussion_r304205961) once we have a preview7 SDK.

Basically revert this commit: https://github.com/aspnet/AspNetCore-Tooling/pull/594/commits/cff58c01ae8151385df156e6eb421d8e613965f9

**Comment by dougbu** [comment]
Unassigned myself because @NTaylorMullen did everything
